packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts

```diff
@@ -126,6 +127,15 @@ export interface ContainerDefinitionOptions {
    */
   readonly command?: string[];
 
+  /**
+   * A list of ARNs in SSM or Amazon S3 to a credential spec (`CredSpec`) file that configures the container for Active Directory authentication.
+   *
+   * We recommend that you use this parameter instead of the `dockerSecurityOptions`. Only the first entry on this array is used. This may be expanded in the future.
```

Comment:
```suggestion
   * We recommend that you use this parameter instead of the `dockerSecurityOptions`.
   * Currently, only one credential spec is allowed per container definition.
```

Reply:
done